src/include/btree.h

```diff
@@ -95,7 +95,11 @@ struct __wt_btree {
 	uint32_t maxleafkey;		/* Leaf page max key size */
 	uint32_t maxleafvalue;		/* Leaf page max value size */
 	uint64_t maxmempage;		/* In-memory page max size */
+	uint32_t maxmempage_image;	/* In-memory page image max size */
 	uint64_t splitmempage;		/* In-memory split trigger size */
+#define	WT_COMPRESS_ADJ		100
+	uint32_t intl_compadjust;	/* Internal page compression adjust */
+	uint32_t leaf_compadjust;	/* Leaf page compression adjust */
```

Comment:
Do we reconcile enough internal pages to justify splitting the calculation up? I'd be inclined to just share a compression ratio.

Reply:
Well, every leaf page reconciliation triggers a reconciliation of its parent, so I would expect us to reconcile more internal pages than leaf pages.

That said, I don't disagree with you: arguing internal pages compress significantly differently than leaf pages requires that block-manager address cookie values make a difference in compression (vs. normal leaf-page values), and I have no data to show that's the case.